packages/api/lib/granules.js

```diff
@@ -62,10 +64,37 @@ function getGranuleProductVolume(granuleFiles = []) {
     .reduce((x, y) => x + y, 0);
 }
 
+// TODO default arguments
+const deleteGranuleAndFiles = async (
```

Comment:
Docstring please!

Reply:
Yup! Absolutely. You caught me mid-refactor. This function is working and tested but I have this toto and some documentation left on my list. Will comment below when that's pushed.

Reply:
Added the docs in this commit:

7c6fb74b267334220afff90d9e7980d0cf5c9ab5

Reply:
👍🏻  re: docstring.  That commit has a comment that's buried, is that still actionable?

Reply:
Nevermind, I see the comment below.    Resolving this thread.